test/functional/page_objects/dashboard_page.ts

```diff
@@ -291,6 +291,24 @@ export class DashboardPageObject extends FtrService {
     });
   }
 
+  public async clearUnsavedChanges() {
+    this.log.debug('clearUnsavedChanges');
+    let switchMode = false;
+    if (await this.getIsInViewMode()) {
+      await this.switchToEditMode();
+      switchMode = true;
+    }
+    await this.retry.try(async () => {
+      // avoid flaky test by surrounding in retry
+      await this.testSubjects.existOrFail('dashboardUnsavedChangesBadge');
+      await this.clickQuickSave();
+      await this.testSubjects.missingOrFail('dashboardUnsavedChangesBadge');
+    });
+    if (switchMode) {
+      await this.clickCancelOutOfEditMode();
+    }
+  }
+
```

Comment:
Moved this out to be a general function since now two test suites are checking for unsaved changes - `panel_titles.ts` and `dashboard_to_dashboard_drilldown.ts`